tests/test_client.py

```diff
@@ -879,12 +879,17 @@ def foo():
 
     # Fields updated when set
     await orion_client.update_flow_run(
-        flow_run.id, flow_version="foo", parameters={"foo": "bar"}, name="test"
+        flow_run.id,
+        flow_version="foo",
+        parameters={"foo": "bar"},
+        name="test",
+        tags=["hello", "world"],
     )
     updated_flow_run = await orion_client.read_flow_run(flow_run.id)
     assert updated_flow_run.flow_version == "foo"
     assert updated_flow_run.parameters == {"foo": "bar"}
     assert updated_flow_run.name == "test"
+    assert updated_flow_run.tags == ["hello", "world"]
```

Comment:
question that maybe could use a test: does this fully overwrite all the tags that are present, or does it append to the tags on the flow run?

Reply:
Absolutely, good call.